### DISCUSSION TITLE
Images offline?

### DISCUSSION THREAD

**Discussion by KoleckOLP** [Tyrrrz/DiscordChatExporter#664] [General]
Would it be possible to download images fo offline use?

**Comment by JaffaCakes118** [comment]
![image](https://user-images.githubusercontent.com/50051038/127327230-9365fb7e-728e-4cbf-8432-c3abad93cc2b.png)

**Reply by KoleckOLP** [reply]
Oh yeah I had to find the drop down menu.
Thank you.
